v8.go

```diff
@@ -24,6 +24,13 @@ var contexts = make(map[uint]*V8Context)
 var contextsMutex sync.RWMutex
 var highestContextId uint
 
+// Error returned if the operation is terminated prematurely via Terminate().
+// NOTE: this error may be returned even when Terminate() is not called on
+// a specific context, as there is no way to tell v8 to terminate only
+// a certain context -- the call is on the isolate level.
+// TODO(avaskys): Move the Terminate() function from the context to the isolate.
+var TerminatedError = errors.New("Operation terminated prematurely")
```

Comment:
nit: in the Go stdlib, the naming conventions for errors are:

* "err" - variables, fields, and values
* "error" - types and functions

This is sometimes hard to keep straight, and there are exceptions, but it is fairly consistent and I like to follow it when possible. Thus this var should be ErrTerminated.